imagedephi/redact/tiff.py

```diff
@@ -100,17 +147,83 @@ def report_missing_rules(self) -> None:
 
     def report_plan(self) -> None:
         print("Tiff Metadata Redaction Plan\n")
-        for tag_value, rule in self.redaction_steps.items():
+        for tag_value, rule in self.metadata_redaction_steps.items():
             print(f"Tiff Tag {tag_value} - {rule.key_name}: {rule.action}")
         self.report_missing_rules()
-
-    def execute_plan(self) -> None:
+        print("Tiff Associated Image Redaction Plan\n")
+        print(f"Found {len(self.image_redaction_steps)} associated images")
+        if len(self.image_redaction_steps):
+            default_rule = list(self.image_redaction_steps.values())[0]
+            print(f"Redaction action: {default_rule.action}")
+
+    def create_new_image(self, ifd: IFD, rule: ImageReplaceRule, temp_dir: Path) -> Path:
+        image = None
+        if rule.replace_with == "blank_image":
+            width = int(ifd["tags"][256]["data"][0])
+            length = int(ifd["tags"][257]["data"][0])
```

Comment:
Are there symbolic constants for `256` and `257`?

Reply:
b536a25